src/index.js

```diff
@@ -175,15 +175,52 @@ VirtualMachine.prototype.loadProject = function (json) {
     this.clear();
     // @todo: Handle other formats, e.g., Scratch 1.4, Scratch 3.0.
     sb2import(json, this.runtime);
-    // Select the first target for editing, e.g., the stage.
-    this.editingTarget = this.runtime.targets[0];
+    // Select the first target for editing, e.g., the first sprite.
+    this.editingTarget = this.runtime.targets[1];
+    // Update the VM user's knowledge of targets and blocks on the workspace.
+    this.emitTargetsUpdate();
+    this.emitWorkspaceUpdate();
+    this.runtime.setEditingTarget(this.editingTarget);
+};
+
+/**
+ * Add a single sprite from the "Sprite2" (i.e., SB2 sprite) format.
+ * @param {?string} json JSON string representing the sprite.
+ */
+VirtualMachine.prototype.addSprite2 = function (json) {
+    sb2import(json, this.runtime, true);
+    // Select new sprite.
+    this.editingTarget = this.runtime.targets[this.runtime.targets.length - 1];
```

Comment:
I think that would make sense. I also think it could return `null` there to indicate there's no one target to set as the editing target. In that case it's up to the thing that imported to decide which target to use?  I think the stage idea might make more sense though.